features/form_settings_tests.feature

```diff
@@ -80,12 +80,12 @@ Scenario: Test the 'Change Language' setting
 	Then I select form "Question Types"
 
 	Then I select "Change Language" menu item
-	Then I see a list that contains all of these items "en,hin"
-	Then I touch the "hin" text
+	Then I see a list that contains all of these items "English,हिंदी"
```

Comment:
Yup, worked fine for me on my machine. 

I did have to make sure that the file was encoded as UTF-8 without a BOM. The first time I added the unicode it defaulted to UTF-8 With BOM and that didn't work until I fixed it.